recipes/libs/alsa-lib.yaml

```diff
@@ -0,0 +1,24 @@
+inherit: [autotools]
+
+metaEnvironment:
+    PKG_VERSION: "1.2.12"
+    PKG_LICENSE: "LGPL-2.1+ (library), GPL-2.0+ (aserver)"
```

Comment:
This smells like we should split the package, shouldn't we?

Reply:
Maybe we do it the other way around: explicitly only call `autotoolsPackageTgt` in the `packageScript`. Debian is not even packaging aserver. So I guess there is no real use of it anyway.

Reply:
I removed that license, because actually I didn't build aserver.